docs/content/en/install/gke/_index.md

```diff
@@ -381,7 +516,35 @@ Take note of the comments in this file. If you have changed the name of your sec
 
 ### Deploy K8ssandra with Helm
 
-With a `values.yaml` file generated which details out specific configuration overrides we can now deploy K8ssandra via Helm.
+With a `values.yaml` file generated, which details specific configuration overrides, we can now deploy K8ssandra via Helm.
+
+If you haven't already, add the latest stable K8ssandra repo:
```

Comment:
"latest stable" sounds odd as those are the names of two different releases right? can it just say "stable"?